src/view/tables/factory/components/FactoryNameCell.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FactoryResData } from 'services/api';
+import { Tooltip, makeStyles, createStyles } from '@material-ui/core';
+import { UnstyledLink } from 'view/components';
+
+export interface FactoryNameCellProps {
+  // TODO: Remove optional `certificates` once we fix backend issues
+  name: FactoryResData['name'];
+  id: FactoryResData['id'];
+}
+
+const useStyles = makeStyles(({ palette }) =>
+  createStyles({
+    link: {
+      textDecoration: 'none',
+      color: 'inherit',
+    },
+    name: {
+      borderBottom: `2px solid ${palette.primary.light}`,
+      cursor: 'pointer',
+      '&:hover': {
+        borderBottom: `2px solid ${palette.primary.main}`,
+      },
+    },
+  }),
+);
+
+export const FactoryNameCell = ({ name, id }: FactoryNameCellProps) => {
+  const classes = useStyles();
+
+  return (
+    <UnstyledLink to={`factories/${id}`}>
+      <Tooltip
+        placement="top"
+        data-testid="factory-cell-tooltip"
+        title="View Factory Profile"
```

Comment:
I don't have this formalized anywhere but I think we should opt to use regular sentence casing for tooltips, e.g.

```suggestion
        title="View factory profile"
```

I'm sure there are places where we aren't doing that currently but I think it would be wise to start the practice.